pulumi-programs/ttl-stacks/README.md

```diff
@@ -1,15 +1,20 @@
 # TTL stacks
 
-The TTL stack processor is a piece of infrastructure that allows you to set a stack tag that triggers automatic destroy via the Pulumi Deployment API after the expiration time passes. This enables temporary development infrastructure.
+The TTL stack processor is a piece of infrastructure that allows you to set a
+stack tag that triggers automatic destroy via the Pulumi Deployment API after
+the expiration time passes. This enables temporary development infrastructure.
```

Comment:
curious, why add all of the explicit line breaks here instead of letting the markdown renderer deal with it? here and in the rest of the README.